server/Entities/Models/JobTitle.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Entities.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace WebAPI.Models
+{
+    public class JobTitle
+    {
+        public int JobTitleId { get; set; }
+        public string JobName { get; set; }
+
+        public ICollection<CelebrityJobTitles> CelebrityJobTitles { get; set; }
```

Comment:
it seems we have some entities for job titles for celebrities, ask @Yaronberg about it